Sources/DistributedSystem/DistributedSystem.swift

```diff
@@ -533,10 +592,10 @@ public class DistributedSystem: DistributedActorSystem, @unchecked Sendable {
 
                     var channel: Channel?
```

Comment:
Local actors also linked with an `ActorInfo` instance. `Channel` in the `ActorInfo` will be optional in the case if it will be in the `ActorInfo`, and such approach would add some additional possibility to have inconsistent `ActorInfo`, when for example `ActorInfo` is `.remoteClient`, but channel is empty.

Reply:
I meant computed property, not stored.